src/statistic.cpp

```diff
@@ -17,20 +17,55 @@
 using namespace regexbench;
 
 static std::map<std::string, size_t>
-make_statistic(const uint32_t sec, const struct ResultInfo& stat,
-               const struct ResultInfo& total);
+make_statistic(const uint32_t sec, const double usec,
+               const struct ResultInfo& stat, const struct ResultInfo& total);
 static struct ResultInfo realtime(std::vector<MatchResult>& results);
 static struct ResultInfo total(std::vector<MatchResult>& results);
 
-void regexbench::statistic(const uint32_t sec,
+void regexbench::statistic(const uint32_t sec, timeval& begin,
                            std::vector<MatchResult>& results, realtimeFunc func,
                            void* p)
 {
   struct ResultInfo stat = realtime(results);
   struct ResultInfo tstat = total(results);
-  std::map<std::string, size_t> m = make_statistic(sec, stat, tstat);
+  double usec = 0.0;
```

Comment:
Wouldn't it be better to make this `size_t`, since it has to be converted into `size_t` in `make_statistic()` anyway?